web/templates/cluster_hana.html.tmpl

```diff
@@ -25,54 +24,51 @@
                         <strong>SID:</strong><br>
-                        <span class="text-muted">{{ .SID }}</span>
+                        <span class="text-muted">{{ index .Cluster.SIDs 0 }}</span>
                     </div>
                     <div class="col-3 mt-5">
                         <strong>SAPHanaSR health state:</strong><br>
-                        {{ $HANAHealthState := $firstNode.HANAHealthState }}
-                        {{- if eq $HANAHealthState  "4" }}
+                        {{- if eq .Cluster.Details.SRHealthState  "4" }}
                             <i class="eos-icons eos-18 text-success">fiber_manual_record</i>
-                            <span class="text-muted">{{ $HANAHealthState }}</span>
-                        {{- else  if or (eq $HANAHealthState "2") (eq $HANAHealthState "3")  }}
+                            <span class="text-muted">{{ .Cluster.Details.SRHealthState }}</span>
+                        {{- else  if or (eq .Cluster.Details.SRHealthState "2") (eq .Cluster.Details.SRHealthState "3")  }}
                             <i class="eos-icons eos-18 text-warning">fiber_manual_record</i>
-                            <span class="text-muted">{{ $firstNode.HealthState }}</span>
-                        {{- else  if or (eq $HANAHealthState "1")  }}
+                            <span class="text-muted">{{.Cluster.Details.SRHealthState }}</span>
+                        {{- else  if or (eq .Cluster.Details.SRHealthState "1")  }}
                             <i class="eos-icons eos-18 text-danger">fiber_manual_record</i>
-                            <span class="text-muted">{{ $HANAHealthState }}</span>
+                            <span class="text-muted">{{ .Cluster.Details.SRHealthState }}</span>
                         {{- else }}
                             -
                         {{- end}}
                     </div>
                     <div class="col-6 mt-5">
                         <strong>HANA secondary sync state:</strong><br>
-                        {{ $secondarySyncState := .Nodes.HANASecondarySyncState }}
                         {{ $badgeClass := "badge-primary" }}
-                        {{- if eq $secondarySyncState "SFAIL" }}
+                        {{- if eq .Cluster.Details.SecondarySyncState "SFAIL" }}
                             {{ $badgeClass =  "badge-danger" }}
                         {{- end }}
-                        <span class="badge badge-pill {{ $badgeClass }} ml-0">{{ $secondarySyncState }}</span>
+                        <span class="badge badge-pill {{ $badgeClass }} ml-0">{{ .Cluster.Details.SecondarySyncState }}</span>
                     </div>
-
                     <div class="col-3 mt-5">
                         <strong>Stonith type:</strong><br>
-                        <span class="text-muted">{{ .Cluster.FencingType }}</span>
+                        <span class="text-muted">{{ .Cluster.Details.StonithType }}</span>
```

Comment:
As a comment `Stonith` was renamed to `Fencing` due the offensive meaning of Stonith (political correctness). That's why currently it is used more. I know that we have stonith all over the places, but good to know why Fence word is being used lately more

Reply:
ok, i will change it!